docs/docs/README.md

```diff
@@ -10,7 +10,7 @@ Pipedream is the fastest way to automate any process that connects APIs. Build a
 The Pipedream platform includes:
```

Comment:
Everything from here down is a change to a broken link, or removing a file tied to old docs.